packages/gestalt/src/ComboBox.jsdom.test.js

```diff
@@ -124,38 +121,37 @@ describe('ComboBox', () => {
       expect(baseElement).toMatchSnapshot();
     });
 
-    it('renders disabled state', () => {
+    it('renders disabled state', async () => {
       const { baseElement } = renderComboBox({ disabled: true });
       expect(baseElement).toMatchSnapshot();
 
       expect(screen.queryByRole('button')).not.toBeInTheDocument();
 
-      fireEvent.click(screen.getByLabelText(LABEL));
+      await userEvent.click(screen.getByLabelText(LABEL));
```

Comment:
we should always try to use userEvent over fireEvent whenever we are able to

Reply:
perfect